dubbo-rpc/dubbo-rpc-dubbo/src/main/java/org/apache/dubbo/rpc/protocol/dubbo/DubboProtocol.java

```diff
@@ -90,15 +90,47 @@
 
         @Override
         public CompletableFuture<Object> reply(ExchangeChannel channel, Object message) throws RemotingException {
-
             if (!(message instanceof Invocation)) {
                 throw new RemotingException(channel, "Unsupported request: "
                         + (message == null ? null : (message.getClass().getName() + ": " + message))
                         + ", channel: consumer: " + channel.getRemoteAddress() + " --> provider: " + channel.getLocalAddress());
             }
 
-            Invocation inv = (Invocation) message;
-            Invoker<?> invoker = getInvoker(channel, inv);
+            final Invocation inv = (Invocation) message;
+            final Invoker<?> invoker = getInvoker(channel, inv);
+            if (isFailoverInvoke(inv)) {
+                // We execute these failover requests in the failover thread pool, as they may be wrong or timed out.
+                //
+                // Failure to do so may result in the business thread pool being quickly filled.
+                failoverExecutor.execute(() -> doService(channel, invoker, inv));
```

Comment:
will there be any scenario where member variable of invocation can be shared across multle thread?

Reply:
Hi, khan, I am very happy to receive your review again. 
:)

I also considered this problem, but at this time, the invoker has not yet started execution, so the variables unique to our thread have not yet taken effect, such as RpcContext.